src/pages/office/cat-a-mod1/office.cat-a-mod1.page.ts

```diff
@@ -206,6 +184,11 @@ export class OfficeCatAMod1Page extends BasePageComponent {
       select(getTests),
       select(getCurrentTest),
     );
+    const testCategory$ = currentTest$.pipe(
+      select(getTestCategory),
+      map(_ => <TestCategory>_),
```

Comment:
?

Reply:
Gets the test category and casts it into the correct type then needed throughout most of the page properties/selectors in the `init`